libs/components/lists/src/lib/modules/sort/sort.component.ts

```diff
@@ -18,6 +24,7 @@ export class SkySortComponent {
    * sort button's `aria-label` attribute to provide a text equivalent for screen readers
    * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
    * The aria label should normally be context sensitive, e.g. "Sort constituents". This is especially true when multiple sort buttons are in close proximity.
```

Comment:
```suggestion
     * Use a context-sensitive label, such as "Sort constituents." Context is especially important when multiple filter buttons are in close proximity.
```

Reply:
Also, I notice that we use "sort buttons" in this property description, but in the new ` listDescriptor` description on the toolbar docs, we use "sort menu." Is this discrepancy OK? Or should we change the terminology in one place or the other?

Reply:
I went with `button` here because it already existed in other places. I'll update the toolbar description to also use `button`.

Reply:
Done